content/docs/architecture_guide/performance.md

```diff
@@ -85,22 +85,21 @@ For many use cases, fixed memory overheads are a primary concern. This may be
 because sandboxed containers handle a low volume of requests, and it is
 therefore important to achieve high densities for efficiency.
 
-{{< graph id="density" url="/performance/density.csv" title="perf.py density --runtime=runc --runtime=runsc" >}}
+{{< graph id="density" url="/performance/density.csv" title="perf.py density --runtime=runc --runtime=runsc" log="true" y_min="100000" >}}
```

Comment:
Does this graph still look reasonable with y_min="100000"? (I didn't render this)

Graphs that don't start at 0 are often misleading, though I imagine the log scale largely mitigates that.

Reply:
Yeah, it looks okay. Went back and forth on the clearest format. Not misleading though.